src/basics.ml

```diff
@@ -178,44 +178,45 @@ let iter : (term -> unit) -> term -> unit = fun action ->
     | Appl(t,u)  -> iter t; iter u
   in iter
 
-(** [iter_meta f t] applies the function [f] to every metavariable of
-   [t], as well as to every metavariable occurring in the type of an
-   uninstantiated metavariable occurring in [t], and so on. *)
-let rec iter_meta : (meta -> unit) -> term -> unit = fun f t ->
-  match unfold t with
-  | Patt(_,_,_)
-  | TEnv(_,_)
-  | Wild
-  | TRef(_)
-  | Vari(_)
-  | Type
-  | Kind
-  | Symb(_)    -> ()
-  | Prod(a,b)
-  | Abst(a,b)  -> iter_meta f a; iter_meta f (Bindlib.subst b Kind)
-  | Appl(t,u)  -> iter_meta f t; iter_meta f u
-  | Meta(v,ts) -> f v; iter_meta f !(v.meta_type); Array.iter (iter_meta f) ts
-
-(** {b NOTE} that {!val:iter_meta} is not implemented using {!val:iter} due to
-    the fact this it is performance-critical. *)
+(** [iter_meta b f t] applies the function [f] to every metavariable of [t],
+   and of the type of every metavariable recursively if [b]. *)
```

Comment:
Here "of" should be "to". Also the comment is not indented correctly.
The `fun b f ->` part can move to the end of previous line.

Reply:
Fixed in a0113be89c8d0a93481b724398cc2c01ac0b4063. For indentation, I am not sure. Please do it yourself.